qa/TL1_jupyter_conda/test_nofw.sh

```diff
@@ -20,7 +20,7 @@ test_body() {
     # test all jupyters except one related to a particular FW,
     # and one requiring a dedicated HW (multiGPU, GDS and OF)
     # optical flow requires TU102 architecture whilst this test can be run on any GPU
-    exclude_files="multigpu\|mxnet\|tensorflow\|pytorch\|paddle\|external_input.ipynb\|numpy_reader.ipynb\|webdataset-externalsource.ipynb\|optical_flow\|python_operator\|#"
+    exclude_files="multigpu\|mxnet\|tensorflow\|pytorch\|paddle\|jax\|external_input.ipynb\|numpy_reader.ipynb\|webdataset-externalsource.ipynb\|optical_flow\|python_operator\|#"
```

Comment:
Do we test this jupyter in any other test?

Reply:
This is just supposed to fix the nightly. I will add a separate PR which will enable the test of this notebook accordingly. I did not want to do it here since there usually are some issues with adding JAX to various jobs in the CI.